geomesa-tools/src/main/resources/leaflet/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>GeoMesa Leaflet Map Viewer</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css"
+          integrity="sha512-Rksm5RenBEKSKFjgI3a41vrjkw4EVPlJ3+OiI65vTjIdo9brlAacEuKOiQ5OFh7cOI1bkDwLqdLw3Zg0cRJAAQ=="
+          crossorigin=""/>
+    <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"
+            integrity="sha512-/Nsx9X4HebavoBvEBuyp3I7od5tA0UzAxs+j83KgC8PU0kgB4XiK4Lfe4y4cgBtaRJQEIFCW+oC506aPT2L1zw=="
+            crossorigin=""></script>
+    <script src="https://cdn.rawgit.com/aheyne/Leaflet.heat/gh-pages/dist/leaflet-heat.js"></script>
```

Comment:
could we host this somewhere more permanent? we could probably add it as part of this PR and then reference the github URL (although that wouldn't work until the PR is actually merged)

Reply:
I'm not sure, it's licensed under BSD 2-Clause https://github.com/aheyne/Leaflet.heat/blob/gh-pages/LICENSE Would that cause problems?

Reply:
oh, that would be ok i think... but instead, you could just link to the original github repo instead of your fork. just make sure you link to a particular release tag so that it doesn't change out from under us